### DISCUSSION TITLE
Commit Worker - Ryzen 3900x - 1080TI

### DISCUSSION THREAD

**Discussion by ksidles** [filecoin-project/lotus#5684] [Show and tell]
**CPU** - Ryzen 3900x
**GPU** - gtx 1080TI
**GPU Driver** - 450.102.04
**Sealing Storage** - Local NVME
**Lotus Version** - 1.5.0
**Memory** - 3400MHZ Non ECC
**Notes** - Dedicated commit worker with 10gb NIC

**Completion Time** = 1h:17m

**Comment by tutorialevideo** [comment]
Hi! can you share full config of your workers?

**Reply by ksidles** [reply]
Do you mean the environmental variables or the physical configuration?

**Comment by tutorialevideo** [comment]
> **CPU** - Ryzen 3900x
> **GPU** - gtx 1080TI
> **GPU Driver** - 450.102.04
> **Sealing Storage** - Local NVME
> **Lotus Version** - 1.5.0
> **Memory** - 3400MHZ Non ECC
> **Notes** - Dedicated commit worker with 10gb NIC
> 
> **Completion Time** = 1h:17m

this is owkring ok?

**Reply by ksidles** [reply]
Yes, this worker performs great.  It can handle any part of the sealing process.  I am currently using it for PC2 & Commit

**Reply by tutorialevideo** [reply]
nice! can you share all your hardware? or you can help up to setup ?

**Reply by ksidles** [reply]
Threadripper 3960x – 128GB Ram – 4TB nvme – 10gb nic (currently runs lotus & lotus-miner)
Ryzen 3900x – 128GB Ram – 3TB nvme – 1080TI – 10gb nic (currently handles PC2 & C2)
Gigabyte g25n-g51 – 4 1060 GPUs – 256GB Ram – 7.68TB nvme – dual intel cpus – 2 10gb nic (waiting on new gpus)
HPE ProLiant DL325 Gen10 Server EPYC 7351P – 512 GB ram (arriving tomorrow) – 8TB nvme (handles all PC1 jobs)
MikroTik Cloud Router Switch Rack-mountable Manageable Switch with Layer 3 Features (CRS317-1G-16S+RM)

**Reply by tutorialevideo** [reply]
Can you share your discord/slack telegram id? i would love to share some info with you.

thank you in advance
